tools/data-migration-overview.md

```diff
@@ -44,28 +44,55 @@ dmctl is the command line tool used to control the DM cluster.
 - Handling the errors during data synchronization tasks
 - Verifying the configuration correctness of data synchronization tasks
 
-## Data synchronization introduction
+## Data synchronization features
 
-This section describes the data synchronization feature provided by Data Migration in detail.
+This section describes the data synchronization features provided by the Data Migration tool.
+
+### Schema and table routing
```

Comment:
add a link to the introduction of the feature?

Reply:
@csuzhangxc Added in the following text.